lisa/tools/xfstests.py

```diff
@@ -277,26 +279,31 @@ def check_test_results(self, raw_message: str, log_path: Path) -> None:
             fail_info += find_patterns_in_lines(
                 raw_message, [re.compile(f".*{fail_case}.*$", re.MULTILINE)]
             )[0][0]
-        self.save_xfstests_log(fail_cases.split(), log_path)
+        self.save_xfstests_log(fail_cases.split(), log_path, test_type)
+        self.node.execute(f"rm -rf {self.get_tool_path()}", sudo=True)
```

Comment:
Make sure the test is runnable, because the tool may not be installed again in next test case.

Reply:
remove this